test_scripts/Defects/5_0/2395_SDL_must_store_OnWayPointChange_internally_and_resend_this_notification_to_each_subscribed_apps.lua

```diff
@@ -0,0 +1,129 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/2395
+--
+-- Description:
+-- SDL must store OnWayPointChange internally and re-send this notification to each subscribed apps
+-- Steps to reproduce:
+-- 1) App_1 and app_2 successfully subscribes on wayPonts-related data
+--    and SDL stores OnWayPointChange (<wayPoints_1>) notification received from HMI
+--    and HMI sends OnWayPointChange (<wayPoints_2>) notification to SDL
+-- 2) App_1 successfully subscribes on wayPonts-related data
+--    and HMI sends OnWayPointChange () notification to SDL
+-- Expected:
+-- 1) SDL must store OnWayPointChange internally and re-send this notification to each subscribed apps
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('user_modules/sequences/actions')
+
+-- [[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local notifParams = {
+wayPoints =
+    {
+        {
+            coordinate = {
+            latitudeDegrees = -90,
+            longitudeDegrees = -180
+            },
+            locationName = "Ho Chi Minh",
+            addressLines = {"182 Le Dai Hanh"},
+            locationDescription = "Toa nha Flemington",
+            phoneNumber = "1231414",
+            locationImage = {
+            value = common.getPathToFileInStorage("icon.png"),
+            imageType = "DYNAMIC"
+            },
+            searchAddress = {
+            countryName = "aaa",
+            countryCode = "084",
+            postalCode = "test",
+            administrativeArea = "aa",
+            subAdministrativeArea = "a",
+            locality = "a",
+            subLocality = "a",
+            thoroughfare = "a",
+            subThoroughfare = "a"      
+            }
+        }  
+    }
+}
+
+--[[ Local Functions ]]
+local function pTUpdateFunc(tbl)
+    local OWgroup = {
+        rpcs = {
+            GetWayPoints = {
+                hmi_levels = {"BACKGROUND", "FULL", "LIMITED"}
+            },
+            SubscribeWayPoints = {
+                hmi_levels = {"BACKGROUND", "FULL", "LIMITED"}
+            },
+            UnsubscribeWayPoints = {
+                hmi_levels = {"BACKGROUND", "FULL", "LIMITED"}
+            },
+            OnWayPointChange =  {
+                hmi_levels = {"BACKGROUND", "FULL", "LIMITED"}
+            }
+        }
+    }
+    tbl.policy_table.functional_groupings["NewTestCaseGroup"] = OWgroup
+    tbl.policy_table.app_policies[config.application1.registerAppInterfaceParams.appID].groups = {"Base-4", "NewTestCaseGroup"}
+    tbl.policy_table.app_policies[config.application2.registerAppInterfaceParams.appID] = tbl.policy_table.app_policies[config.application1.registerAppInterfaceParams.appID]
```

Comment:
@TMyza Please avoid long sequences of table field data receiving in simple cases. 
There is proposed solution here:
```
local testGroups = { "Base-4", "NewTestCaseGroup" }
tbl.policy_table.app_policies[config.application1.registerAppInterfaceParams.appID].groups = testGroups
tbl.policy_table.app_policies[config.application2.registerAppInterfaceParams.appID].groups = testGroups
tbl.policy_table.app_policies[config.application3.registerAppInterfaceParams.appID].groups = testGroups
```